.github/workflows/guide.yml

```diff
@@ -5,7 +5,7 @@ on:
     branches:
       - master
     tags:
-      - v0.*
+      - v[0-9]+.*
```

Comment:
The dot in the regex matches any symbol, if you need to match literally the dot, you need to escape it?

Reply:
```suggestion
      - v[0-9]+\.*
```

Reply:
Ah yes, I forgot the regex must match the whole version string, and furthermore my suggestion is broken.